phys/module_diag_misc.F

```diff
@@ -159,6 +217,74 @@ SUBROUTINE diagnostic_output_calc(                                 &
 
    INTEGER :: idump
 
+!-----------------------------------------------------------------
+! Compute physics tendency accumulations in a given time window
+
+   IF ( acc_phy_tend .gt. 0 ) THEN
+
+!  !$OMP PARALLEL DO   &
+!  !$OMP PRIVATE ( ij )
+   DO ij = 1 , num_tiles
+
+      IF (mod(curr_secs2, history_interval*60.) == 0.) THEN
+         WRITE(outstring,*) 'Reseting accumulation to 0'
```

Comment:
Resetting
No debug level for this?

Reply:
@dudhia Could remove it or make it dependent on debug level.

Reply:
We can use wrf_debug instead.

Reply:
Is it a useful print or can we just remove it?